rpc/params/doc.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package params contains the definition of the request and response parameters for the Juju API Server and
+// Juju API callers (which includes the user-facing clients but also the agents and the controller itself).
+// When introducing a change to a facade, to maintain compatibility with a previous version of the facade, copy the existing parameter struct
```

Comment:
I'd move the "When introducing..." bit to a separate paragraph below the "API messages are encoded in JSON..." one.